build.js

```diff
@@ -81,10 +81,11 @@ if (!REPLAY_LOCAL_DRIVER_DIR) {
   fs.unlinkSync(driverJSON);
 }
 
-let driverString = "";
+let driverString = [];
 for (let i = 0; i < driverContents.length; i++) {
-  driverString += `\\${driverContents[i].toString(8)}`;
+  driverString.push(`\\${driverContents[i].toString(8)}`);
 }
+driverString = driverString.join("");
```

Comment:
For the record, if hex escapes work here instead of this current octal escape, this would also be an option:
```
const driverString = driverContents.toString("hex").replace(/\d\d/g, "\\X$&");
```